src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/security/VideoBookingAccess.kt

```diff
@@ -11,11 +11,15 @@ import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.service.UserType
  */
 fun checkVideoBookingAccess(externalUser: User, booking: VideoBooking) {
   if (externalUser.isUserType(UserType.EXTERNAL)) {
+    if (!booking.prisonIsEnabledForSelfService()) {
```

Comment:
The only thing to be wary of here is appointments (at the moment) are lazily loaded on bookings so this must be called in the context of a `@Transactional` annotation.

Personally I'd be happy for the appointments to be eager on VideoBooking.